estate/models/EstateProperty.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "The estate_property table stores real estate property details and metadata for transaction management."
+    active = fields.Boolean(default=False)
```

Comment:
```suggestion
    _description = "The estate_property table stores real estate property details and metadata for transaction management."

    active = fields.Boolean(default=False)
```
Bad readability, fields needs to be bifurcate with a extra new line differently from static variable `_name` and `_description` 

Also what do you understand by the field `active`? What will be the default value for a Boolean field if I don't pass it here?

Reply:
active basically indicates the weather property should be listed or not. default value for a Boolean is False.